src/transformers/Lichess.js

```diff
@@ -0,0 +1,27 @@
+const { URL } = require('url');
+
+const includesSomeOfArray = (string, array) =>
+  array.some(item => string.includes(item));
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+
+  return (
+    (host == 'lichess.org' || host == 'www.lichess.org') &&
+    !includesSomeOfArray(pathname, [
+      '/embed/',
+      '/learn',
+      '/practice',
+      '/study',
+      '/training',
+      '/tv/',
```

Comment:
Since it is possible a Lichess game can have the URL like [https://lichess.org/tv1342Jy](https://lichess.org/tv1342Jy), it would result in incorrect result.

Note, the above URL is made up, but that situation is probable.

Reply:
@MichaelDeBoey could you please comment on that, before I make changes for `Array.includes`?

Reply:
Good catch!

Please add a test for https://lichess.org/tv123abc56de to make this clear in our tests too 🙂

Reply:
Done in the latest commit.